internal/storage/postgres/sensor/create.go

```diff
@@ -82,12 +82,22 @@ func (r *SensorRepository) InsertSensorData(ctx context.Context, data []*entitie
 }
 
 func (r *SensorRepository) createEntity(ctx context.Context, sensor *entities.Sensor) (string, error) {
-	return r.store.CreateSensor(ctx, &sqlc.CreateSensorParams{
-		ID:        sensor.ID,
-		Status:    sqlc.SensorStatus(sensor.Status),
+	id, err := r.store.CreateSensor(ctx, &sqlc.CreateSensorParams{
+		ID:     sensor.ID,
+		Status: sqlc.SensorStatus(sensor.Status),
+	})
+	if err != nil {
+		return "", err
+	}
+
+	if err := r.store.SetSensorLocation(ctx, &sqlc.SetSensorLocationParams{
```

Comment:
I am not quite sure why the sensor location is set after the sensor is created? Is

Reply:
Because of the geometry attribute.